setup.py

```diff
@@ -249,17 +249,7 @@ def build_extensions(self):
     long_description=Path("README.rst").read_text(encoding="utf-8"),
     long_description_content_type="text/x-rst",
     license="PSF",
-    license_files=[
-        "LICENSE/LICENSE",
-        "LICENSE/LICENSE_AMSFONTS",
-        "LICENSE/LICENSE_BAKOMA",
-        "LICENSE/LICENSE_CARLOGO.txt",
-        "LICENSE/LICENSE_COLORBREWER",
-        "LICENSE/LICENSE_QT4_EDITOR",
-        "LICENSE/LICENSE_STIX",
-        "LICENSE/LICENSE_TORICK",
-        "LICENSE/Solarized.txt",
-    ],
+    license_files=list(map(lambda p: str(p), Path("LICENSE").glob('*'))),
```

Comment:
Small style thing: can this just be `[str(p) for p in Path("LICENSE").glob('*')]` ?

Reply:
I would agree, nice catch, but per comment below we'll actually need to move this to `setup.cfg` if we want this to be useful and actually used by `wheel`.